docs/protos/go.md

```diff
@@ -59,22 +56,49 @@ import (
 
 func main() {
 	jobclient := mgmtv1alpha1connect.NewJobServiceClient(
-		http.DefaultClient,
-		os.Getenv("NEOSYNC_API_URL"),
+		newHttpClient(map[string]string{ // create an instance of a newHttpClient
+			"Authorization": fmt.Sprintf("Bearer %s", os.GetEnv("API_KEY")), // pass in the API_KEY through an environment variable
+		}),
+		os.GetEnv("API_URL"), // pass in the API_URL through an environment variable
 	)
 
-	_, err := jobclient.CreateJobRun(context.Background(), connect.NewRequest(&mgmtv1alpha1.CreateJobRunRequest{
-		JobId: "<neosync-job-id>",
-	}))
-	if err != nil {
-		panic(err)
+	// rest of code to call an API in the JobServiceClient goes here
+	// ...
+}
+
+func newHttpClient(
+	headers map[string]string,
+) *http.Client {
+	return &http.Client{
+		Transport: &headerTransport{
+			Transport: http.DefaultTransport,
+			Headers:   headers,
+		},
+	}
+}
+
+type headerTransport struct {
+	Transport http.RoundTripper
+	Headers   map[string]string
+}
+
+func (t *headerTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	if req.Header == nil {
+		req.Header = http.Header{}
 	}
+	for key, value := range t.Headers {
+		req.Header.Add(key, value)
+	}
+	return t.Transport.RoundTrip(req)
 }
+
 ```
 
-## Go SDK Packages
+## Getting started
```

Comment:
small thing but I wonder if this should be above the authentication section?

Reply:
traditionally authentication comes first, since you install -> authenticate -> then do something

Reply:
yea it's just a little weird because it shows a full example, and then it goes into the "getting started" section that talks all about the makeup of the packages.
I think that was why I previously had the auth in its own section separate.